tests/sit-stand-sit_v0.0.1.md

```
@@ -0,0 +1,9 @@
+# sit-stand-sit_v0.0.1
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)